src/features/shared-link-settings-modal/SharedLinkSettingsModal.scss

```diff
@@ -1,6 +1,16 @@
 @import '../../styles/variables';
 
 .shared-link-settings-modal {
+    /** title and classifiction label */
+    .bdl-SharedLinkSettingsModalTitle {
```

Comment:
These new classes don't need to be nested, you can have all of them at the top level.
Only the bdl-is-disabled (and other similar states or modifiers) should be &.nested since they should never exist on their own all alone.